client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigationMenu/InvestigationMenuStyles.ts

```diff
@@ -0,0 +1,18 @@
+import { makeStyles } from '@material-ui/styles';
+const ITEM_HEIGHT = 48;
+
+const useStyles = makeStyles({
+    menu: {
+        maxHeight: ITEM_HEIGHT * 4.5,
+        width: '23ch',
+        borderRadius: '10px',
+        flip: false,
+        right: '2vw!important',
+        top: '12vh!important',
```

Comment:
has to be important?

Reply:
yup, or else its overriden by mui menu styles